.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -3,21 +3,31 @@ name: New release template
 about: Template for creating new releases of Durable Functions
 title: ''
 labels: ''
-assignees: cgillum
+assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
 ---
 
-As part of a release, the following items need to be taken care of. For any items that are not applicable, they should be crossed out using the `~~ ~~` markdown syntax with an explanation.
-
-- [ ] Run private scale tests and ensure no regressions
-- [ ] Publish updated versions of [DurableTask.Core](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.Core/) and [DurableTask.AzureStorage](https://www.nuget.org/packages/Microsoft.Azure.DurableTask.AzureStorage/) to nuget.org
+**Prep Release: (assigned to:)**
+_Due: <two-businessdays-before-release>_
 - [ ] Merge all features and fixes into the `master` branch
-- [ ] Update .NET API docs at https://azure.github.io/azure-functions-durable-extension
-- [ ] Publish signed version of [Microsoft.Azure.WebJobs.Extensions.DurableTask](https://www.nuget.org/packages/Microsoft.Azure.WebJobs.Extensions.DurableTask/) to nuget.org
-- [ ] Update samples to point to the latest nuget packages
+- [ ] Publish updated versions of DurableTask.Core and DurableTask.AzureStorage to app-service MyGet feed
+- [ ] Increment extension version
+- [ ] Publish signed version of Microsoft.Azure.WebJobs.Extensions.DurableTask to app-service MyGet feed
+
+**Validation (assigned to: )**
+_Due: <1-businessdays-before-release>_
+- [ ] Run private performance tests and ensure no regressions
+- [ ] Smoke test Functions V1 and Functions V3 .NET apps
+- [ ] Smoke test JavaScript and Python apps
+
```

Comment:
A recent release had us accidentally increase the size of our package by *3x*. In fact, that was my fault :-P .
I would like to add a step here where we also check for package size.

```suggestion
**Validation (assigned to: )**
_Due: <1-businessdays-before-release>_
- [ ] Run private performance tests and ensure no regressions
- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release
- [ ] Smoke test Functions V1 and Functions V3 .NET apps
- [ ] Smoke test JavaScript and Python apps

```

Reply:
I'm also not sure that 1 business day is enough to give us time to react if the validation steps go wrong. I don't feel strongly though, we can iterate on this.

Reply:
I guess this goes into when we want to code freeze. The release prep step probably takes ~1-2 hours, so they could happen on the same day, but the release prep needs to happen _before_ validation. If we are cool with everything merging to master by 3 days ahead of time, we can have validation happen in the second half 3 days a head of time, or early in 2 days before. We then have a pretty substantial buffer built in.

Reply:
Shifting release prep to 3 days before, and validation step to 2 days before. Seems like a good place to start.